modules/api/php/endpoints/candidates.class.inc

```diff
@@ -241,8 +241,14 @@ class Candidates extends Endpoint implements \LORIS\Middleware\ETagCalculator
                 $pscid,
                 $project->getId()
             );
-        } catch (\LorisException | \InvalidArgumentException $e) {
-            return new \LORIS\Http\Response\JSON\BadRequest($e->getMessage());
+        } catch (\ConflictException $e) {
```

Comment:
why not just make this catch `\InvalidArgumentException`?

Reply:
Because it can throws a lot of Exceptions types. createNew doesn't try/catch so everything bubbles up.

Reply:
Also, `InvalidArgumentException` is used for the 404